Dockerfile

```
@@ -1,9 +1,4 @@
-FROM debian:10-slim
-# We're stuck on Debian 10 (which will go EOL around 2022-08) because
-# Debian 11 upgrades GNU Make to v4.3, which breaks Buildroot's build system.
-#
-# Upstream Buildroot has fixes for this (try grepping their CHANGES file for
-# "4.3"), but those commits aren't currently in this fork.
+FROM debian:stable-slim
```

To reduce unpleasant surprises in the future, I would consider keeping this pinned to the major version:

```suggestion
FROM debian:11-slim
```

(from https://hub.docker.com/_/debian)

From what I remember from working on #132, when `stable-slim` rolled over from `10-slim` to `11-slim`, it caused downstream problems that weren't trivial to debug.